aspnetcore/blazor/performance/developer-tools-profiling.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: ASP.NET Core Blazor WebAssembly performance profiling and diagnostic counters
+title: ASP.NET Core Blazor WebAssembly developer tools performance profiling and diagnostic counters
 author: guardrex
-description: Learn about performance profiling and diagnostic counters in ASP.NET Core Blazor WebAssembly apps.
+description: Learn about developer tools performance profiling and diagnostic counters in ASP.NET Core Blazor WebAssembly apps.
```

Comment:
It's "browser developer tools" a normally hidden panel in the browser.
It doesn't support "diagnostic counters" only "sampling profiler" via code instrumentation.